src/collections/Breadcrumb/Breadcrumb.js

```diff
@@ -21,12 +21,12 @@ function Breadcrumb(props) {
   const dividerJSX = <BreadcrumbDivider icon={icon}>{divider}</BreadcrumbDivider>
   const sectionsJSX = []
 
-  sections.forEach(({ text, ...restSection }, index) => {
-    const key = `${text}-${index}`
```

Comment:
This is a happy medium i think.  All generated component props should allow passing an object for props.  If a user duplicates text in their breadcrumb (or any other component), then they will have to specify their own unique keys.